spec/types/union.spec.ts

```diff
@@ -28,4 +28,15 @@ describe('union', () => {
       }
     );
   });
+
+  it('supports nullable union types', () => {
```

Comment:
Since this one is handled in a different manner based on versions it makes sense to add one more test for 3.1 as well?